dnd5e/module/item/entity.d.ts

```diff
@@ -1,38 +1,19 @@
-interface Labels {
-  spell: {
-    level: string;
-    school: string;
-    components: Record<DND5e.SpellComponent, string>;
-    materials: string;
-  };
-
-  feat: {
-    featType: string;
-  };
-
-  equipment: {
-    armor: string;
-  };
-
-  activation: {
-    target: string;
-    range: string;
-    duration: string;
-    recharge: string;
-  };
-
-  actionType: {
-    damage: string[];
-    damageTypes: string[];
-  };
-}
-
-type AnyLabel = Labels[keyof Labels];
+type Items5e = {
+  weapon: Item.Data<Item5e.Data.Weapon> & { type: 'weapon' };
```

Comment:
This is my fault for leaving it in the preliminary stage but I'd like this to be accessed somehow else. Item5e['weapon'] is awkward when we already have namespaces. Maybe Item5e.Data.Weapon or whatever else works... not sure what to do with Item5e.Data.